src/components/SidePanel/NavGroup/FilterNavGroup/FilterGroup/index.tsx

```diff
@@ -78,11 +97,20 @@ const FilterGroup: React.FC<FilterGroupProps> = ({
           <StyledTextInput
             aria-label="Edit filter name"
             aria-labelledby="Edit filter name"
+            autoFocus
             data-cy="edit-filter-name"
-            onChange={(e) => setNewFilterName(e.target.value)}
+            errorMessage={!isValid ? validationMessage : ""}
```

Comment:
nit. I think it might be slightly more intuitive if we reversed the condition
```suggestion
            errorMessage={isValid ? "" : validationMessage}
```